pkg/services/authz/rbac/store/permission_query.sql

```diff
@@ -1,4 +1,4 @@
-SELECT p.kind, p.attribute, p.identifier, p.scope FROM {{ .Ident .PermissionTable }} as p
+SELECT p.action, p.kind, p.attribute, p.identifier, p.scope FROM {{ .Ident .PermissionTable }} as p
```

Comment:
this will require some snapshot updates -- run the tests locally, and it will create them

Reply:
Yeah I updated them but @IevaVasiljeva pointed out that we don't need the action its part of the sql filter. We just need to check scope after